src/midisequencer.cpp

```diff
@@ -99,6 +109,13 @@ namespace midisequencer{
     {
         return getc(static_cast<std::FILE*>(fp));
     }
+    static bool is_loop_start(uint_least32_t msg) {
+        // If the message matches the de facto standard MIDI loop instruction:
+        // Which is a Control Change on any channel with Value 111
```

Comment:
Is this tested on channels different than 1 from RPG_RT?

What about multiple control changes, does RPG_RT pick the first or the last?

Reply:
RPG_RT definitely allows any channel. I will verify the behaviour of RPG_RT when there’s multiple loop starts found sometime this weekend 👍

Reply:
Confirmed our behaviour matches RPG_RT here: they'll (and we) use the last loop start found (when sorting by time)